### DISCUSSION TITLE
Stand for dispaly

### DISCUSSION THREAD

**Discussion by Darkstar2k2003** [CircuitSetup/Time-Circuits-Display#31] [Ideas]
How is everyone displaying theirs? It would be nice to have a 3d printed stand designed for this.

**Comment by blevinweb** [comment]
I have mine on my desk and display by having it sit on a laptop stand. This way I can adjust the height and positioning. I have one similar to this and it looks great on there: https://a.co/d/0jrpmGT

**Reply by Darkstar2k2003** [reply]
Thanks! I got one of those and it works great!

**Comment by realA10001986** [comment]
I bought this:
https://www.amazon.com/RIWUCT-Adjustable-Ergonomic-Ventilated-Compatible/dp/B0B1L6R9NN/
stand, cut off the two "arms" in the front and had someone make me a piece of acrylic glass (with a cut-out for the keypad) which I taped on top. The clock is held in place by neodym magnets which I taped on the inside of the enclosure as well as the top plate.

(This is my dev kit clock, by far not the most beautiful one)

![IMG_9725a](https://user-images.githubusercontent.com/76924199/216148186-db2b542b-23d5-453c-8e3e-381a1a963b54.jpg)
![IMG_9726a](https://user-images.githubusercontent.com/76924199/216148215-fcb3a142-e800-40a5-b543-44a0086ec0f7.jpg)
![IMG_9727a](https://user-images.githubusercontent.com/76924199/216148227-4c8a7008-a71e-43da-8ba8-492f3a8feff5.jpg)

**Reply by Lxmcps28** [reply]
I have a newer design enclosure that has holes in the back for mounting it in a vehicle like the original prop. However, I also have the same problem with the unit being bottom-heavy and not staying in place when set on the edge of a table. The keypad enclosure wants to pull the entire unit down, since it is the heaviest part of the unit. Mine was displayed this way with a weight on top to hold it in place. Unfortunately the weight shifted at some point, causing the unit to fall to the floor and cracking the enclosure housing. Luckily I had the USB cable connected using a magnetic connector instead of the usual plug-in type, so the cable disconnected before the unit hit the floor, sparing any possible port damage. Customer support knew about the issue, but hasn't come up with a solution, or offered to replace the damaged enclosure. 
How much did it cost to buy the laptop stand, acrylic and magnets to secure the unit? 
![Enclosure mishap](https://user-images.githubusercontent.com/124397295/216694011-d08f78b2-d734-4da9-ab29-92f508e9d9c3.jpg)

**Reply by realA10001986** [reply]
Well, the laptop stand is available at Amazon for $22; the acrylic glass was around $15 (plus shipping), the magnets I also got via Amazon, around $15. I actually bought magnets that were too strong, so I broke one in half. I am using one 30x10x5mm on each side taped to the unit, and one 15x10x10mm piece on each side taped to the acrylic glass, You can lift the entire construction by the clock and type on the keypad without supporting it from the back, that's how strong these Neodym thingies are.


PS: Do yourself a favor and buy the aluminum (or aluminium for everyone outside the US) version, you'll never take another look at the plastic one afterwards ;)

**Comment by Ricky3Nut** [comment]
That looks really cool. Good idea and thanks for the inspiration. I might do something similar and modify the stand I have. 

Though I'm not sure about the position of the keypad when it's attached to the bottom of the display so I'll probably do something slightly different.



Get BlueMail for Android

**Comment by Lxmcps28** [comment]
I decided on a simpler setup; the holes on the sides of the enclosure in
the back will be used to secure brackets on the inside, which will be
fastened to a piece of plywood
that should provide enough weight to hold the unit in place. I still want
to have it be able to sit on the edge of the table I am going to display it
on, and the wood will be hidden
inside the enclosure. If the wood alone isn't enough, I can always add some
weight to it. Better than having something sitting on top of the enclosure.

**Comment by doc1011** [comment]
I've made my own on a shelf using an adapted 3d printed phone stand and some clips. I've made a custom mini digital speedo and wired in the time travel button. Works great for me. 

![B021E438-C26F-47ED-B4D1-3650800CC8A1_1_105_c](https://user-images.githubusercontent.com/42011383/218192685-c7365d0a-34a4-4bb5-b927-188917960dae.jpeg)

**Comment by Lxmcps28** [comment]
Looks great. How did you secure the unit to the phone stand?

**Reply by doc1011** [reply]
it just rests on it with a bit of bluetack

**Comment by Lxmcps28** [comment]
Had to have the external housing replaced, as the unit I have decided to
fall off the place I had it displayed, cracking the side. It was replaced
at no cost since the imbalance issue had been known for some time, but a
solution is still in the works. For now, I'm going to use a piece of wood
inside the enclosure to give it enough weight to offset the keypad housing,
which is heavier. An aluminum enclosure is way more than I want to spend
right now, and they are currently unavailable until the backorders are
filled. I still like the look of the one I have.
